vendor/bat-native-ads/src/bat/ads/internal/tokens/redeem_unblinded_payment_tokens/redeem_unblinded_payment_tokens.cc

```diff
@@ -48,12 +48,17 @@ void RedeemUnblindedPaymentTokens::set_delegate(
 
 void RedeemUnblindedPaymentTokens::MaybeRedeemAfterDelay(
     const WalletInfo& wallet) {
-  if (is_processing_ || retry_timer_.IsRunning()) {
+  if (is_processing_ || timer_.IsRunning() || retry_timer_.IsRunning()) {
```

Comment:
Fixes an issue discovered when adding the unit tests where multiple timers would be queued up, however only one of those timers would fire at a time, so everything was still working as expected. To prove this, I removed this line and re-ran the tests.